tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -29,21 +29,27 @@ or logos must follow Microsoft's [Trademark & Brand Guidelines](https://www.micr
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 `;
 
+// Packages which do not publish on NPM
 function packageShouldBePrivate(name: string): boolean {
     // allow test packages to be packaged
     if (name.startsWith("@fluid-internal/test-")) {
         return false;
     }
 
     return (
-        name === "root" || // minirepo roots
-        name.startsWith("@fluid-internal"));
+        name === "root" // minirepo roots
```

Comment:
```suggestion
        name === "root" // monirepo roots
```

My coining the term "minirepo" never really caught on :P

Reply:
monorepo?